docs/troubleshooting/troubleshooting.aad.md

```diff
@@ -53,7 +53,7 @@ nmi-pv8ch                              1/1     Running   1          15m
 
 ### Issue in MIC Pod
 For `mic` pod, we will need to find the leader. An easy way to find the leader is by looking at the log size. Leader pod is the one that is actively working.
-1. MIC pod communicates with Azure to assign the identity to the AKS nodes. If there are any issues in outbound connectivity, MIC can report TCP timeouts.
+1. MIC pod communicates with Azure to assign the identity to the AKS nodes. If there are any issues in outbound connectivity, MIC can report TCP timeouts. Check your NSGs, UDRs and Firewall to make sure that you allow outbound traffic to Azure.
```

Comment:
nit: everything talked here is on Azure :)

Reply:
I can change it to `Azure Resource Manager(ARM)` to make more sense.